ui/app/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -63,15 +63,15 @@ export default function AwaitingSwap ({
   const approveTxParams = useSelector(getApproveTxParams)
   const tradeTxParams = useSelector(getSwapsTradeTxParams)
   const currentCurrency = useSelector(getCurrentCurrency)
-  const conversionRate = useSelector(conversionRateSelector)
+  const usdConversionRate = useSelector(getUSDConversionRate)
 
   const [timeRemainingExpired, setTimeRemainingExpired] = useState(false)
   const [trackedQuotesExpiredEvent, setTrackedQuotesExpiredEvent] = useState(false)
 
-  let feeinFiat
+  let feeinUnformattedFiat
   if (usedQuote && tradeTxParams) {
-    const renderableGasFees = getRenderableGasFeesForQuote(usedQuote.gasEstimateWithRefund || usedQuote.averageGas, approveTxParams?.gas || '0x0', tradeTxParams.gasPrice, currentCurrency, conversionRate)
-    feeinFiat = renderableGasFees.feeinFiat?.slice(1)
+    const renderableGasFees = getRenderableGasFeesForQuote(usedQuote.gasEstimateWithRefund || usedQuote.averageGas, approveTxParams?.gas || '0x0', tradeTxParams.gasPrice, currentCurrency, usdConversionRate)
```

Comment:
Formatting is both unnecessary and diverges from what we want in our analytics, which is an unformatted decimal number, which is what `rawNetworkFees` gives us.